specs/UI/Snomed.spec

```diff
@@ -0,0 +1,19 @@
+# Snomed Diagnosis
+
+## Doctor should be able to add Diagnosis and save which is coming from snomed ct server
+
+tags: snomed, ui
+
+* Login to Bahmni snomed location "General Ward" as a "receptionist"
+* Create new patient with random details and start an OPD visit
+* Logout and Login to Bahmni snomed location "General Ward" as a "doctor"
+* Open clinical tab
+* Open "Diagnoses" Tab
+* Doctor adds random snomed diagnosis "consultation/diagnosis/disease"
+* Verify random snomed diagnosis is added to openmrs database
```

Comment:
Change this to "Verify random snomed diagnosis saved is added to openmrs database with required metadata"